packages/pods-core/src/v2/ConfigFileUtils.ts

```diff
@@ -67,4 +73,44 @@ export class ConfigFileUtils {
     }
     return fPath;
   }
+
+  static createExportConfig(opts: { required: string[]; properties: any }) {
+    return {
+      type: "object",
+      required: ["podId", "podType", "exportScope", ...opts.required],
+      properties: {
+        podId: {
+          description: "configuration ID",
+          type: "string",
+        },
+        description: {
+          description: "optional description for the pod",
+          type: "string",
+          nullable: true,
+        },
+        exportScope: {
+          description: "export scope of the pod",
+          type: "string",
+        },
+        podType: {
+          description: "type of pod",
+          type: "string",
+        },
+        ...opts.properties,
+      },
+    };
+  }
+
+  static getConfigSchema(podType: PodV2Types): any {
```

Comment:
Let's not return `any` - at the very least, we should be able to do `JSONSchemaType<any>`?

Reply:
Chatted offline to resolve this one later.